microsoftTeams - microsoft-teams-message-send-to-chat message_type ar… (#39761)
demisto/content: Packs/MicrosoftTeams/ReleaseNotes/1_5_26.md

#### Integrations

##### Microsoft Teams

> - The **microsoft-teams-message-send-to-chat** command will now only supports sending messages from message type: message

Reviewer: ```suggestion
- The **microsoft-teams-message-send-to-chat** command now only supports sending messages of type: ***message***.
```
